scripts/src/main/resources/scripts/command/graalvm

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# $1: optional setup
+function doSetup() {
+  installDir="${DEVON_IDE_HOME}/software/extra/graalvm"
+  if doIsMacOs
+  then
+    GRAALVM_HOME="${DEVON_IDE_HOME}/software/extra/graalvm/Contents/Home"
+  else
+    GRAALVM_HOME="${installDir}"
+  fi
+
+  doDevonCommand java setup silent
+  if [ ! -d "${installDir}" ]
+  then
+    local version="${GRAAL_VERSION:-22.0.0.2}"
+    local javaVersion="${EXTRA_JAVA_VERSION}"
+    if [ -z "${javaVersion}" ]
+    then
+      javaVersion="${JAVA_VERSION}"
+    fi
+    if [ -n "${javaVersion}" ]
+    then
+      doVersionCompare "${javaVersion}" "11u0"
+      if [ "${?}" == 2 ]
+      then
+        # Java version is lower than 11
+        doConfirmWarning "GraalVM requires Java 11 or newer.\nHowever, [EXTRA_]JAVA_VERSION is ${javaVersion} - in this setup GraalVM can not work.\nPlease update JAVA_VERSION or set EXTRA_JAVA_VERSION in your settings/devon.properties!"
+      fi
+    fi
+    doInstall "-" "${installDir}" "graalvm" "${version}"
+    echo -e "export GRAALVM_HOME=${GRAALVM_HOME}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
```

Comment:
This is not really robust:
* what if `GRAAL[VM]_VERSION` is changed and the user reruns `devon graalvm setup`? This should upgrade graalvm - currently it IMHO does not. If you would change the `if` statement to make it work, the user would get multiple entries of `export GRAALVM_HOME=...` in his devon.properties. The same already happens now if you delete `software/graalvm` after installation and rerun `devon graalvm setup`.
* The user is in control of the file `conf/devon.properties`. He can edit this file. In case he adds a line like
`#end` to the end of the file that is not terminated with a newline, you would IMHO append it to `#endexport GRAALVM_HOME=....` and it would not work. Simply prefix with `\n` should solve this as you are already use `-e` option.

Please also add `GRAALVM_HOME` to documentation (`variables.asciidoc`).

Reply:
Ok, I found a solution. GraalVM will also install other versions that are installed via `GRAALVM_VERSION` specified. Export Statement will only added to `devon.properties` if not added before. `\n` is added to the export statement.
`GRAALVM_HOME` is also added to the documentation.